practical/jython/template_run_macro_for_another_user_on_dataset.jy

```diff
@@ -0,0 +1,341 @@
+# -----------------------------------------------------------------------------
+#  Copyright (C) 2018 University of Dundee. All rights reserved.
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ------------------------------------------------------------------------------
+
+# This Jython script shows how to load a given plane from OMERO,
+# load OMERO rois and add them to the manager and run a ImageJ macro.
+# In that script, the analysis is done on behalf of another user by a person
+# with more privileges e.g. analyst.
+# All planes can be retrieved or only the planes corresponding to a specific timepoint
+# More details about restricted privileges can be found at
```

Comment:
this sentence should be above the previous one, just after ``In that script,... e.g. analyst.`` because it describes the same topic